react/src/component/MovieComponent.js

```diff
@@ -0,0 +1,61 @@
+import React, {useEffect} from 'react';
+import {useParams, useHistory} from 'react-router-dom';
+import { Image, Jumbotron} from 'react-bootstrap';
+import {getMovieById} from '../store/actions/MovieActions';
+import {useDispatch, useSelector} from 'react-redux';
+import {movieSelector} from '../store/selectors/MovieSelector';
+import Comment from './Comment';
+
+export default function MovieComponent() {
+
+    const dispatch = useDispatch();
+    const history = useHistory();
+
+    let { id } = useParams();
+
+    useEffect(() => {
+        dispatch(getMovieById(id));
+    },[]);
+
+    const movie = useSelector(movieSelector());
+
+    return(
+        <div>
+            {movie ? 
+            (
+                <div>
+                    <button className='m-1' onClick={() => history.goBack()}><i className="far fa-arrow-left"></i>  Back</button>
```

Comment:
```suggestion
                    <button className='m-1' onClick={history.goBack}><i className="far fa-arrow-left"></i>  Back</button>
```
Mozda ce samo ovo raditi?